build-logic/src/main/kotlin/com.hedera.hashgraph.repositories.gradle.kts

```diff
@@ -36,4 +32,21 @@ repositories {
     maven {
         url = uri("https://us-maven.pkg.dev/swirlds-registry/maven-develop-snapshots")
     }
+    maven {
```

Comment:
Not sure if the below is still needed. But all repositories that were potentially used are now here in one place.